src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CollectionsPagesSteps.java

```diff
@@ -63,4 +63,25 @@ public void verifyCMSFAQLink(final String cmsfaqLink) {
         assertTrue(link.contains(cmsfaqLink), "Expect " + cmsfaqLink + " to be found in " + link);
     }
 
+    /**
+     * Get user to the Collections page.
+     */
+    @Given("^I am on the Collections page$")
+    public void iAmOnCollectionsPage() {
+        driver.get(url + "#/collections/sed");
+        WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(CollectionsPages.mainContent(driver)));
+    }
+
+    /**
+     * Download SED Details.csv file.
+     * @param fileName expected file name
+     */
+    @When("^I download the \"([^\"]*)\" file$")
+    public void downloadSurveillanceActivityFile(final String fileName) {
```

Comment:
Function name suggests surveillance. First line of the method looks like SED. Step definition doesn't care. I'm not sure what this is supposed to do